apps/dev-api/domains/common/utils/sms.js

```diff
@@ -0,0 +1,53 @@
+const conf = require('@open-condo/config')
+
+const { developmentClient, productionClient } = require('@dev-api/domains/common/utils/serverClients')
+
+class FakeSMSAdapter {
+    async sendMessage (phone, message) {
+        console.log(JSON.stringify({ adapter: 'Fake SMS Adapter', phone, message }))
+    }
+}
+
+class CondoSMSAdapter {
+    #client
+
+    constructor (serverClient) {
+        this.#client = serverClient
+    }
+
+    async sendMessage (phone, message) {
+        await this.#client.sendMessage({ phone }, message)
+    }
+}
+
+class SMSAdapter {
+    #internalAdapter = null
+
+    constructor (type = 'fake') {
+        switch (type) {
+            case 'condo-dev':
+                this.#internalAdapter = new CondoSMSAdapter(developmentClient)
+                break
+            case 'condo-prod':
+                this.#internalAdapter = new CondoSMSAdapter(productionClient)
+                break
```

Comment:
Why not just `dev` and `prod`? IMO `condo-` prefix is redundant here

Reply:
The reason for that is dev portal API not works like usual app (dev stand uses dev condo, prod uses prod condo).

In reality dev-api uses both stands at the same time. Prefix condo needed to specify that `condo` is a provider of SMS.

(Maybe we'll want to create a separate SMSC for this, so it will became SMSC-dev and SMSC-prod or smth like this)